### ISSUE TITLE
using browser "back" function sometimes provokes 406 not acceptable

### ISSUE THREAD

**Issue by paregorios** [isawnyu/pleiades-gazetteer#96]
Steps to reproduce:

- Visit ```/places/433211```
- Select the first link under "Locations", labeled "location of course of Volturnus fl. (modern)"
- After ```/places/433211/location-of-course-of-Volturnus-fl``` has loaded, select the browser back button
- Contemplate the message below (note that doing a browser refresh/reload at this point will banish the 406 message and get you the place page back)

![unacceptable](https://cloud.githubusercontent.com/assets/263285/13763177/3f07710a-ea12-11e5-9e11-cee99980e849.jpg)

**Comment by davisagli** [comment]
There's some magic I wasn't aware of in the apache configuration that goes to a "conneg" view that negotiates which view to serve based on the request's Accept header, if that header does not contain 'html' or equal '*/*'. Will need to figure out why the back button triggers this.

**Comment by paregorios** [comment]
Ah, yes. That apache magic is about [content negotiation in the context of linked data](https://www.w3.org/TR/swbp-vocab-pub/#negotiation).

**Comment by davisagli** [comment]
Fixed.

This was a bit of a doozy. It was happening like this:
1. You load the main place view. The place_map.js javascript tries to do an AJAX request to load the json for the baseline. It gets this URL from a `<link rel="baseline-where">`, but since you aren't on a working copy, that is not present and it ends up with a URL of "", which jquery interprets as wanting to load the current URL. So it does that with an Accept: application/json header. That's not one of the content types that the content negotiation script knows about, so it returns the 406 message.
2. When you navigate back to the place URL, the browser shows the last response for that URL, which is that 406. It does this even though the `conneg` script sets a Cache-control: no-cache header.

Fixed by a) fixing place_map.js to not try to load baseline json when there's no baseline, and b) setting up the place view and conneg script so they'll set a Vary: Accept response header, so that downstream caches realize that they need to pay attention to the Accept request header.

I also removed the Cache-control: no-cache header because it shouldn't be necessary now that the Vary header is set correctly. At some point we might want to configure varnish to do some normalization of the Accept header so that we don't store multiple copies of a place view just because different browsers send slightly different headers.
